pom.xml

```diff
@@ -398,6 +414,37 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.pitest</groupId>
+                <artifactId>pitest-maven</artifactId>
+                <version>${pit-plugin.version}</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.pitest</groupId>
+                        <artifactId>pitest-junit5-plugin</artifactId>
+                        <version>${pit-junit-plugin.version}</version>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <targetClasses>
+                        <!-- we not want to include also WaitException, Utils classes etc. -->
+                        <param>io.strimzi.test.container.Strimzi*</param>
```

Comment:
What about `KafkaContianer` and `KafkaServiceVersion`? They are not needed as well?

Reply:
Hmm KafkaContianer is just interface but fair enough if we add some default methods there then it would make sense. I will update it. And for  `KafkaServiceVersion` I didn't want to include and focus on main classes but if you think it would make sense I can add it. :)

Reply:
Maybe more explanation in comment would be good, I am fine with any change you prefer as SME.